newIDE/app/src/MainFrame/TabsTitlebar.js

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import MenuIcon from '../UI/CustomSvgIcons/Menu';
 import IconButton from '../UI/IconButton';
 import ElementWithMenu from '../UI/Menu/ElementWithMenu';
-import ThemeContext from '../UI/Theme/ThemeContext';
+import ThemeContext from '../UI/Theme/GDevelopThemeContext';

```

Comment:
could rename this to GDevelopThemContext too

Reply:
Updating this.